### ISSUE TITLE
`ruff.logLevel` doesn't apply the log level

### ISSUE THREAD

**Issue by ErikBuis** [charliermarsh/ruff-vscode#161]
I configured the `ruff.logLevel` my `settings.json` file to be `"info"`, but the extension keeps reporting all detected problems as warnings (yellow underlines). It seems like all other logLevel values also do not change the way these problems are displayed.

For clarity, I would like the extension to behave similarly to Flake8, where all problems are reported as information (blue underlines). For Flake8, I would use:
```json
"python.linting.flake8CategorySeverity.E": "Information"
"python.linting.flake8CategorySeverity.W": "Information"
"python.linting.flake8CategorySeverity.F": "Information"
```

Additional information:
    OS: Ubuntu 22.04.2 LTS
    Ruff VS Code extension version: v2023.9.10740409 (Pre-Release), but the issue is also reproducible on the Release Version.
    VS Code version: 1.76.2

**Comment by charliermarsh** [comment]
Ah, we don't support severities like that right now, though we probably will in the future. The `logLevel` is related to how much logging the Ruff outputs in the VS Code logs, _not_ the level of violations that are reported to the user.

**Comment by charliermarsh** [comment]
Related: https://github.com/charliermarsh/ruff/issues/1256
